modules/nw-ingress-controller-configuration-proxy-protocol.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * ingress/configure-ingress-operator.adoc
+
+[id="nw-ingress-controller-configuration-proxy-protocol_{context}"]
+= Configuring the PROXY protocol for an Ingress Controller
+As a cluster administrator, you can use the PROXY protocol to enhance communication between the Ingress Controller and an external load balancer. When you publish an Ingress Controller with a load balancer, the connections that the Ingress Controller receives have the load balancer's source address, instead of the original client address. The PROXY protocol enables a load balancer to preserve the original client addresses and include them when forwarding connections to an Ingress Controller, which can then use the original client addresses for logging.
+
+You can configure an Ingress Controller to use the PROXY protocol when the endpoint publishing stratey type is `HostNetwork` or `NodePortService`. Set the `spec.endpointPublishingStrategy.nodePort.protocol` or `spec.endpointPublishingStrategy.hostNetwork.protocol` field to `PROXY`.
+
+This feature is not supported in cloud deployments. This restriction is because when {product-title} runs in a cloud platform, and an IngressController specifies that a service load balancer should be used, the Ingress Operator configures the load balancer service and enables the PROXY protocol based on the platform requirement for preserving source addresses.
+
+
+[WARNING]
+====
+To prevent connection failure, configure both the Ingress Controller and the load balancer to use the PROXY protocol.
+====
+
+.Prerequisites
+* You created an Ingress Controller.
+
+.Procedure
+. Configure the Ingress Controller API to use the PROXY protocol:
++
+.. Configure the Ingress Controller resource:
++
+[source,terminal]
+----
+$ oc -n openshift-ingress-operator edit ingresscontroller/default
+----
++
+.. Edit the YAML file:
++
+.Sample `nodePort` configuration to `PROXY`
+[source,yaml]
+----
+  spec:
+    endpointPublishingStrategy:
+      nodePort:                 <1>
+        protocol: PROXY
+----
+<1> Instead of setting the `nodePort` subfield, you can also set the `hostNetwork` subfield.
```

Comment:
The correct way to define the `PROXY` configuration is as below:
- For `hostnetwork` strategy:
```
   endpointPublishingStrategy:
     hostNetwork:
       protocol: PROXY            
     type: HostNetwork
````
- For `Nodeport` strategy:
```
 endpointPublishingStrategy:
   nodePort:
     protocol: PROXY            
   type: NodePortService
```

Reply:
@quarterpin Ok I reworked this example a bit to hopefully make this clearer. Thanks for your feedback!